resources/scripts/generate-email-template-if-no-recent-changes.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
For the record, I initially started to implement this as part of the groovy script, but I didn't like the idea of coupling the implementation with the groovy/pipeline so I ended up using a script, that can run anywhere

In addition, the first attempt was with the `gh list prs` closed and opened ones, but the log of the file with the elastic-stack version should be enough to know if it has been updated recently.